inferno-framework/tutorial-test-kit
# FI-3166: Step 6 solution upgrade

## Summary
This PR updates the Inferno template version in step-6-solution branch. It includes the solution to step-6 of the tutorial.

## Testing
Run locally with Docker:
- Run `./run.sh` in this repository
- Navigate to `http://localhost`
- Run test kit with the following inputs:
    - FHIR Base Server URL: `https://inferno.healthit.gov/reference-server/r4`
    - Standalone Client ID: `SAMPLE_PUBLIC_CLIENT_ID`
    - access_token: `SAMPLE_TOKEN`
    - Patient ID: `85`
- You should get the following results:

![image](https://github.com/user-attachments/assets/e103957a-a1e5-4548-b3f8-d3274c89ccac)

NOTE: The 2.02 Test fails because the Patients in the reference server no longer conform to USCore 3.1.1. This is expected.

Also run with Ruby.
